library/foundation/src/main/java/com/epmedu/animeal/foundation/spacer/Spacer.kt

```diff
@@ -0,0 +1,18 @@
+package com.epmedu.animeal.foundation.spacer
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.Dp
+
+@Composable
+fun HSpacer(height: Dp) {
+    Spacer(modifier = Modifier.height(height))
+}
+
+@Composable
+fun WSpacer(width: Dp) {
+    Spacer(modifier = Modifier.width(width))
+}
```

Comment:
Please rename them to HeightSpacer and WidthSpacer

Reply:
Or SpacerHeight and SpacerWidth as @VEZE suggested

Reply:
Done, renamed to WidthSpacer.